### ISSUE TITLE
the files number of TUT 2017 dataset

### ISSUE THREAD

**Issue by Zth9730** [microsoft/Pengi#12]
The Table 2 in the paper says that TUT 2017 contains 6.3k files and includes training, testing, and validation sets. But when I download TUT 2017, there are only 312*15=4680 files and there is only development set. May I ask why there are 6.3k number of files here and how it is divided into training testing validation set?
![image](https://github.com/microsoft/Pengi/assets/32243340/04433511-09b7-4548-b60f-f8b1b3626354)

**Comment by soham97** [comment]
Hi @Zth9730, TUT 2017 has a total of 6.3k files. It's divided into development and evaluation and can be obtained from the below links:
- Development set: https://zenodo.org/records/400515
- Evaluation set: https://zenodo.org/records/1040168

Table 2 in the paper shows full dataset statistics. For Pengi's zero-shot evaluation, we use only the evaluation set to ensure our numbers are comparable with other zero-shot and supervised benchmarks. I hope this helps!

**Comment by Zth9730** [comment]
Thanks a lot !!!
